rfc/3/index.md

```diff
@@ -0,0 +1,158 @@
+> image plane ("yx") and images are stacked along the other (anisotropic) axis
+> ("z"), the spatial axes SHOULD be ordered as "zyx".
+
+And:
+
+> Each "datasets" dictionary MUST have the same number of dimensions and MUST
+> NOT have more than 5 dimensions.
+
+These restrictions are actively preventing users from converting existing
+datasets to NGFF. For example, Zeiss .czi datasets [may contain][czi format
+dimensions] dimensions such as H, I, and V to store different phases,
+illumination directions, or views respectively. To say nothing of synthetic data
+that may contain "artificial" dimensions such as principal components or axes of
+other dimensionality reduction-techniques from many images.
+
+## Proposal
+
+This document proposes removing any restrictions on the number of dimensions
+stored in NGFF arrays. Additionally, it removes restrictions on the names and
+types of included dimensions.
+
+## Prior art and references
+
+All of the above removals are part of the draft proposed [transformations
+specification][trafo spec], with one exception: the draft currently specifies
+that a dataset may only have up to three spatial axes. However, this limitation
+is [not set in stone][space dims comment] and could be removed, partly to
+improve backwards compatibility.
+
+## Stakeholders
+
+Who has a stake in whether this RFC is accepted?
+
+* Facilitator: Josh Moore (OME)
+* Reviewers:
+  - John Bogovic (HHMI Janelia Research Campus): lead author of draft
+  [transformations specification proposal][trafo spec]
+  - Will Moore (OME): maintainer of ome-zarr-py library
+  - Norman Rzepka (Scalable Minds): maintainer of zarrita
+  - ... Your name here?
+* Consulted:
+  - ...
+* Socialization:
+  - image.sc: https://forum.image.sc/t/ome-ngff-update-postponing-transforms-previously-v0-5/95617/2
+
+## Implementation
+
+A draft implementation is [already in progress][implementation].
+
+## Backwards Compatibility
+
+Since this proposal only removes restrictions, these changes are backwards
+compatible at the file level: v0.4 files would transparently be v0.5 files if
+this proposal is approved.
+
+Any readers or writers that proactively checked the dimension restrictions
+(number of dimensions, dimension names, dimension types) would need to remove
+those checks. However, this should be a small amount of work in most cases.
+
+## Forward Compatibility
+
+A draft proposal for [coordinate transformations][trafo spec] already includes
+most of the changes proposed here, so we envision that this RFC is compatible
+with future plans for the format. The proposal does currently limit the number
+of dimensions of type "space" to at most 3, but that limit [could be
+removed][space dims comment]. If this RFC is approved, the transformation
+specification would need to be updated to reflect this. However, that is an easy
+change and there seems to be sufficient support in the community for this idea.
```

Comment:
Do we need to talk about that (stalled) PR at all? I don't see why it's relevant here

Reply:
It's relevant because it speaks to the forward compatibility of this RFC — ie it is in line with existing proposals for the format. That the PR is stalled is not really relevant — it's stalled because of minor details (e.g. array order) that don't have a bearing on this PR. Based on the discussion, other aspects, and certainly the ones relevant to this RFC, have quite broad consensus.

Reply:
in that case isn't it sufficient to just state that there are no known conflicts with other active proposals?